src/auth/utils.ts

```diff
@@ -43,14 +43,19 @@ import { Auth } from './auth'
 import { validateIsNewSsoUrl, validateSsoUrlFormat } from './sso/validation'
 import { openUrl } from '../shared/utilities/vsCodeUtils'
 
-export async function promptForConnection(auth: Auth, type?: 'iam' | 'sso') {
+// TODO: Look to do some refactoring to handle circular dependency later and move this to ./commands.ts
+export const showConnectionsPageCommand = 'aws.auth.showConnectionsPage'
```

Comment:
maybe `./src/shared/vscode/commands2.ts` is a place where these names could live